src/test/java/StringClassTest.java

```diff
@@ -0,0 +1,77 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@DisplayName("String 클래스에 대한 학습 테스트")
+public class StringClassTest {
+
+    @Test
+    @DisplayName(" \"1,2\" 문자열은 ',' 구분자로 분리되어야 한다.")
+    void split_01() {
+        //given
+        String word = "1,2";
+
+        //when
+        String[] result = word.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?